<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>render_widgets</class>
 <widget class="QWidget" name="render_widgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>446</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeadlineRenderJob</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <widget class="QFrame" name="frame_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Render Layer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="render_layer_lv">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="MakePreviewCheckBox">
              <property name="text">
               <string>Make Preview</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="OverWriteLatestVersionCheckBox">
              <property name="text">
               <string>Overwrite empty frames with the latest version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_10">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>start frame</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="cutin_frame_spb">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999999</number>
               </property>
               <property name="value">
                <number>101</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="start_frame_spb">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>999999999</number>
               </property>
               <property name="value">
                <number>101</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>end frame</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="end_frame_spb">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99999999</number>
               </property>
               <property name="value">
                <number>101</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="cutout_frame_spb">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maximum">
                <number>9999999</number>
               </property>
               <property name="value">
                <number>101</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="maximumSize">
                <size>
                 <width>99</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>output size % : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="output_size_percent_spb">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>width  :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="output_size_width_lb">
                   <property name="text">
                    <string>2048</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Height :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="output_size_height_lb">
                   <property name="text">
                    <string>1668</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string> &lt;&lt; </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sg_resolution_pushButton">
               <property name="minimumSize">
                <size>
                 <width>165</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Get ShotGrid Resolution</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string> Camera : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cam_comboBox">
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Priority: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="priority_spbox">
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>50</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_9">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>frame for Task: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="frame_for_task_spb">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="mr_frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Renderer : </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="renderer_lb">
                 <property name="text">
                  <string>renderer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5_empty">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_13">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Region Division</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="render_division_spb">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="send_job_btn">
     <property name="text">
      <string>Send Job</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../reference/img/img.qrc"/>
 </resources>
 <connections/>
</ui>
